src/index.js

```diff
@@ -174,7 +174,8 @@ export default function App() {
             content.projects.miramar.summary.paragraph_two, images.content.miramar_cover);
         let ecuador = projectInfoSection(content.projects.ecuador.header, content.projects.ecuador.summary.paragraph_one,
             content.projects.ecuador.summary.paragraph_two, images.content.ecuador_cover);
-        let sanctuary_farm = projectInfoSection(content.projects.sanctuary_farm.header, content.projects.sanctuary_farm.summary.paragraph_one, content.projects.sanctuary_farm.summary.paragraph_two, images.content.sanctuary_farm_cover);
+        let sanctuary_farm = projectInfoSection(content.projects.sanctuary_farm.header, content.projects.sanctuary_farm.summary.paragraph_one, 
+            content.projects.sanctuary_farm.summary.paragraph_two, images.content.sanctuary_farm_cover);
```

Comment:
Instead of using a blank string in the second paragraph.  We should use the `null` keyword to make it clear that it is unused.